README.md

```diff
@@ -107,67 +157,101 @@ that runs at a much lower price than normal instances. However, Compute
 Engine might terminate (preempt) these instances if it requires access 
 to those resources for other tasks; default: `false`
 
-### `public_key_path`
+### `service_account_name`
 
-Path to the public half of the ssh key that will be deployed to
-`~username/.ssh/authorized_keys`; see also [`username`](#username) below.
+The name of the service account to use when enabling account scopes. This
+is usually an email-formatted service account name created via the "Permissions"
+tab of the GCP console.
 
-### `region`
+This is ignored unless you specify any `service_account_scopes`.
+
+Default: "default"
+
+### `service_account_scopes`
 
-Region in which to launch instances.  `region` is defined as the part
-prior to the second hyphen in an availability zone's name; e.g. in
-`us-central1-b`, the region is `us-central1`.  Specifying region but
-not `zone_name` allows kitchen-gce to avoid launching instances into a
-zone that is down for maintenance.  If `any` is specified, kitchen-gce
-will select a zone from all regions.  Default: `us-central1` (lowest
-cost region); valid values: `any`, `asia-east1`, `europe-west1`,
-`us-central1`
+An array of scopes to add to the instance, used to grant additional permissions
+within GCP.
+
+The scopes can either be a full URL (i.e. `https://www.googleapis.com/auth/devstorage.read_write`)
+or the short-name (i.e. `devstorage.read_write`), but aliases (i.e. `storage-rw`) are not permitted.
+
+See the output of `gcloud compute instances create --help` for a full list of scopes.
 
 ### `tags`
 
 Array of tags to associate with instance; default: `[]`
 
-### `username`
+### `use_private_ip`
 
-Username test-kitchen will log into instance as; default: `ENV['USER']`
+Boolean indicating whether or not to connect to the instance using its
+private IP address. If `true`, kitchen-google will also not provision
+a public IP for this instance. Default: `false`
 
-### `zone_name`
+### `wait_time`
 
-Location into which instances will be launched.  If not specified, a
-zone is chosen from available zones within the [`region`](#region).
+Amount of time, in seconds, to wait for any API interactions. Default: 600
 
-## Example
+### `refresh_rate`
 
-An example `.kitchen.yml` file using kitchen-gce might look something
-like this:
+Amount of time, in seconds, to refresh the status of an API interaction. Default: 2
 
-```ruby
+### Transport Settings
+
+Beginning with Test Kitchen 1.4, settings related to the transport (i.e. how to connect
+to the instance) have been moved to the `transport` section of the config, such as the
+username, password, SSH key path, etc.
+
+Therefore, you will need to update the transport section with the username configured
+for the SSH Key you imported into your project metadata as described in the "SSH Keys"
+section above.  For example, if you are connecting as the "chefuser", your .kitchen.yml
+might have a section like this:
+
+```yaml
+transport:
+  username: chefuser
```

Comment:
I should note that Fog will do some key deployments for you. This rewrite no longer uses Fog and instead uses native GCE functionality.